protobuf/traits/energy_storage.proto

```diff
@@ -135,6 +135,10 @@ message EnergyLevel {
 
     // A measure of how far the device can travel with its current energy level.
     float distance_km = 4;
+
+    // Battery voltage.
+    // The relationship between voltage and energy level / percentage charge is device-specific.
+    optional float voltage = 5;
```

Comment:
Do you think it's worth keeping the fields consistent, aka remove the `optional` marker? I think zero is not ambiguous with absent/unsupported when combined with any other field. And if all fields are zero, then the only ambiguity is around units, the semantics are equal: the storage is empty.

If we need to make the fields `optional` in the future, we'll need to do it for all of them anyway.

Reply:
I lean towards making it optional because 

1. 0 is in range for this field, and can be considered distinct from 'no voltage available'
2. Our newer APIs tend to use optional in this scenario
3. Implicit fields are recommended against by the proto3 language guide

Basically I want to adding more things that we need to make a breaking change to later.

Reply:
OK, though I think inconsistency has a higher cost than it might first appear to have.